bundles/framework/admin-announcements/resources/locale/en.js

```diff
@@ -0,0 +1,41 @@
+Oskari.registerLocalization(
+{
+    "lang": "en",
+    "key": "admin-announcements",
+    "value": {
+        "title": "Admin-announcements",
+        "desc": "",
+        "flyout": {
+            "title": "Admin-announcements"
+        },
+        "tile": {
+            "title": "A: Announcements"
+        },
+        "addNewForm": "New Announcement",
+        "delete": "Delete",
+        "save": "Save",
+        "cancel": "Cancel",
+        "newAnnouncement": {
+            "title": "Title",
+            "content": "Content",
+            "date-range": "Date range",
+            "show-popup": "Show pop-up",
+            "save": "Save"
```

Comment:
"Save" is localized twice here. Could both functionalities requiring it use the same one?